### DISCUSSION TITLE
为什么要手工引入antd/dist/antd.css啊

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [ant-design/ant-design#34155] [Q&A]
官方例子上，需要手工导入antd/dist/antd.css？请问一下是新的写法？如果是新的引入方式，从哪个版本开始的啊？为什么要手工引入啊？

**Comment by KAROTT7** [comment]
antd/dist/antd.css 包含所有组件样式，记得 v3 版本就开始要在入口文件中引入

**Comment by Airkro** [comment]
https://github.com/ant-design/ant-design/issues/23988

这里已经讨论过了
